### DISCUSSION TITLE
i logged into client's vercel account now i can't login into my account

### DISCUSSION THREAD

**Discussion by cezarneaga** [vercel/community#3556] [Account Issue] [answered]
### Question

Hi vercel,

my client gave me their email credentials to login and connect the repo i built for them and setup deployments.
now i cannot login into my account using github. this used to work before. but not anymore.

is this use case not allowed anymore on vercel? or is it something else?

thanks,
Cezar

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the accounts. They will be able to access more information to help you figure out what is going wrong.
